gbeaudry321/ser401-NASA-psyche-mission
# Us213 control rig gbeaudry

Updated astronaut with new mesh.

This doesn't work well. Something is broken in the animations, but I want to get the astronaut into the dev branch so I don't have to find it again if I have time to get back to it after we complete the VR experience. 

To Test: 
 - Open Psyche_Asteroid_Surface
 - Change GameMode Override to BP_ThirdPersonGameMode
 - Play level

Expected Results:
 - Astronaut of appropriate proportions should appear.
 - - Movement is buggy, camera doesn't properly keep up with astronaut
 - - Jumping doesn't work properly
 - Pressing "m" should bring up a map
 - Pressing "t" should bring up a teleport menu
 - Pressing "v" should switch to first person view
